apps/web/components/v2/apps/IntegrationListItem.tsx

```diff
@@ -32,7 +33,8 @@ function IntegrationListItem(props: {
     <ListItem
       expanded={!!props.children}
       className={classNames(
-        "my-0 flex-col border transition-colors duration-500 first:rounded-t-md last:rounded-b-md",
+        props.seperated ? "rounded-md" : "first:rounded-t-md last:rounded-b-md",
```

Comment:
Now we correctly show the rounded corners for separated apps. This how it currently looks:
![Screenshot 2022-10-13 at 11 33 08](https://user-images.githubusercontent.com/30310907/195560646-6386ae25-db48-491d-8b0a-d1e8568f11f6.png)

Reply:
Instead of doing it only for automation categories, why not use the separate approach in all the categories. Don't think, designwise it would be a problem and it would be consistent.
Plus, AppSettings are planned for all the apps no matter which category